docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md

```diff
@@ -0,0 +1,95 @@
+# 配置文件说明
+
```

Comment:
Do you need to add some short description for this reference guide?

Reply:
The parameters are described in the table.